modules/nlu/src/backend/predict-pipeline.ts

```diff
@@ -314,7 +313,7 @@ function electIntent(input: PredictStep): PredictStep {
     predictions.length &&
     predictions[0].name !== NONE_INTENT &&
     predictions[0].confidence < 0.4 &&
-    _.get(input, 'oos_predictions.label') === 'out'
+    input.oos_predictions?.confidence > 0.3
```

Comment:
Same magic number here, also could you extract the one of 0.4 just before ? Would make sense to have those as a constant at the top of the file